Buildings/package.mo

```diff
@@ -229,6 +229,17 @@ The following <b style=\"color:blue\">new components</b> have been added
 to <b style=\"color:blue\">existing</b> libraries:
 </p>
 <table class=\"releaseTable\" summary=\"summary\" border=\"1\" cellspacing=\"0\" cellpadding=\"2\" style=\"border-collapse:collapse;\">
+
+<tr><td colspan=\"2\"><b>Buildings.Fluid.HeatExchangers</b>
+    </td>
+</tr>
+<tr><td valign=\"top\">Buildings.Fluid.HeatExchangers.ThermalWheels.Latent
+    </td>
+    <td valign=\"top\">Package of models for air-to-air heat recovery devices.<br/>
```

Comment:
Change it to thermal wheels, as a plate heat exchanger is also an air-to-air heat recovery device.